test/unit/cli.spec.js

```diff
@@ -183,6 +183,18 @@ describe('cli', () => {
       expect(options.addedFiles).to.deep.equal(['a1.js', 'a2.js'])
       expect(options.changedFiles).to.deep.equal(['ch1.js', 'ch2.js'])
     })
+
+    it('should error on args with underscores', () => {
+      let expectedException
```

Comment:
wanna use 'var' here for consistency?  or use 'const' below for options?

Reply:
done